app/views/report_mailer/tables_report.html.erb

```diff
@@ -24,14 +24,24 @@
     <tbody>
       <% rows.each do |row| %>
         <tr>
-          <% row.each do |cell| %>
-            <td <%= cell.is_a?(Numeric) ? 'class=table-number' : nil %> >
-              <% if cell.is_a?(Float) && report.float_as_percent? %>
-                <%= number_to_percentage(cell * 100, precision: report.precision || 2) %>
-              <% else %>
-                <%= number_with_delimiter(cell) %>
-              <% end %>
-            </td>
+          <% row.each_with_index do |cell, index| %>
+            <% if index == 0 %>
+              <th scope="row" <%= cell.is_a?(Numeric) ? 'class=table-number' : nil %> >
+                <% if cell.is_a?(Float) && report.float_as_percent? %>
+                  <%= number_to_percentage(cell * 100, precision: report.precision || 2) %>
+                <% else %>
+                  <%= number_with_delimiter(cell) %>
+                <% end %>
+              </th>
+            <% else %>
```

Comment:
instead of hardcoding an index, what if we had two separate loops?

```erb
<% first, *rest = row %>
<th ..... >
</th>
<% rest.each do |cell| %>
<% end %>
```

Reply:
It looks like we may not want that after all. The design and ticket are different.

Reply:
or maybe we can keep the loop as is, but find a way to not have to repeat the stuff inside the cell? so maybe

```erb
          <% row.each_with_index do |cell, index| %>
            <%
              tag_attrs = {
                class: cell.is_a?(Numeric) ? 'table-number' : nil,
              }
              if index == 0
                tag = 'th'
                tag_attrs[:scope] = 'row'
              else
                tag = 'td'
                tag_attrs = {}
              end
            %>
            <%= content_tag(tag, tag_attrs) do %>
              <% if cell.is_a?(Float) && report.float_as_percent? %>
                <%= number_to_percentage(cell * 100, precision: report.precision || 2) %>
              <% else %>
                <%= number_with_delimiter(cell) %>
              <% end %>
           <% end %>
          <% end %>
```

Reply:
cool, I didn't know about that destructuring pattern. Thanks